core/domain/exp_services.py

```diff
@@ -625,6 +815,16 @@ def _create_exploration(
 
     This is because _save_exploration() depends on the rights object being
     present to tell it whether to do strict validation or not.
+
+    Args:
+        committer_id: str. The id of the user who made the commit.
+        exploration: exploration. The exploration domain object.
```

Comment:
(Also, note that the reason we use Exploration is because that's what you get when you do `type(...)` on the argument. It refers to the Exploration class defined in exp_domain.py.)